storybooknative/src/Accessibility.js

```diff
@@ -0,0 +1,30 @@
+// @flow strict
```

Comment:
This file should probably be renamed `accessibility.flow.js` or similar, as one would believe this is a component called Accessibility.